core/domain/exp_services.py

```diff
@@ -1329,7 +1726,19 @@ def _is_suggestion_handled(thread_id, exploration_id):
 
 
 def _create_change_list_from_suggestion(suggestion):
-    """Creates a change list from a suggestion object."""
+    """Creates a change list from a suggestion object.
+
+    Args:
+        suggestion: Suggestion domain object from which a change list
+                    is generated.
+
+    Returns:
+        cmd: cmd edit state property of the exploration domain object.
```

Comment:
```
list(dict). The changelist corresponding to the given suggestion domain object.
```

(I think you can drop the rest, since then you're starting to describe the implementation. The above is all a caller really wants to know.)